maven-plugins/javadoc-maven-plugin/src/main/java/io/helidon/build/javadoc/JavadocMojo.java

```diff
@@ -236,6 +237,20 @@ public class JavadocMojo extends AbstractMojo {
     @Parameter(property = "helidon.javadoc.pomExcludes")
     private List<String> pomExcludes = List.of();
 
+    /**
+     * File name patterns.
+     * List of include file names.
+     */
+    @Parameter(property = "helidon.javadoc.fileIncludes", defaultValue = "*.java")
+    private List<String> fileNameIncludes = List.of();
```

Comment:
Can we use the same name, in configuration and property.
I like `fileIncludes` over `fileNameIncludes`.